<launch>

    <arg name="duckiebot" default="duckiebot" />
    <arg name="resolution_m" default="0.005" doc="map grid cell width in m/cell, assuming 
                                                a quadratic grid cell structure." />
    <arg name="vis_update_s" default="0.01" doc="visualisation update rate [s]" />
    <arg name="vis_point_rad" default="0.005" doc="trajectory visualization point rad [m]." />
    <arg name="vis_car_width" default="0.07" doc="car rect visualization width [m]." />
    <arg name="vis_car_height" default="0.05" doc="car rect visualization height [m]. " />

    <node name="itype_control" pkg="duckietown-intnav" type="itype_control.py"  output="screen">
        <param name="duckiebot" value="$(arg duckiebot)"/>
    </node>

    <node name="imap_server" pkg="duckietown-intnav" type="imap_server.py"  output="screen" required="true">
        <param name="duckiebot" value="$(arg duckiebot)"/>
        <param name="resolution_m" value="$(arg resolution_m)"/>
        <param name="vis_update_s" value="$(arg vis_update_s)"/>
        <param name="vis_point_rad" value="$(arg vis_point_rad)"/>
        <param name="vis_car_width" value="$(arg vis_car_width)"/>
        <param name="vis_car_height" value="$(arg vis_car_height)"/>
    </node>

</launch>
